conda_build/api.py

```diff
@@ -394,11 +394,20 @@ def update_index(dir_paths, config=None, force=False, check_md5=False, remove=Fa
         with open(current_index_versions) as f:
             current_index_versions = yaml.safe_load(f)
 
+    if os.path.lexists(key_interface):
+        # TODO: Update location of authenticate module when it's published in
+        #         conda.common.
+        from .authenticate import keyfiles_to_keys
+        priv_key, _ = keyfiles_to_keys(key_interface)
```

Comment:
It's a little goofy, but since there's a private key file and a public key file, `keyfiles_to_keys()` takes a filename prefix and tries to load <prefix>.pri and <prefix>.pub.  So we can either feed it both or just read in the private key file and then call `private_key_from_bytes()`.

Reply:
Oh ya! I like that second way a lot better